README.md

```diff
@@ -25,23 +25,28 @@ Knead Recipes is an app designed to take the hassle and confusion out of what’
 # Features
 
 ## Sign in using Google
-
+![Landing Page](https://i.imgur.com/igWKviA.jpg)
+![Google Sign in](https://i.imgur.com/U6g4wqf.png)
 ---
 ## Set up user profile
-
-
+![Profile Edit](https://i.imgur.com/9cdQ3kf.jpg)
```

Comment:
Since these are just screenshots, we made a couple more small changes - like we can now update profile pictures as well and quickly styled this page so the font matches.

Reply:
new images have been uploaded